chapters/synchronous.tex

```diff
@@ -1043,6 +1044,33 @@ \subsection{Sub-Clock Partitioning}\label{sub-clock-partitioning}
 \end{lstlisting}
 \end{example}
 
+\begin{example}
+Forbidding systems of equations involving different sub-clocks means that the following is forbidden:
+\begin{lstlisting}[language=modelica]
+  Real a;
+  //Real x=a+z;
+  Real y=superSample(a+z, 2);
+  Real z;
+equation
+  a+z = sample(time, Clock(1,100));
+  0 = subSample(y, 2)+a;
+\end{lstlisting}
+Here \lstinline!a! and \lstinline!z! are part of one sub-clock, and \lstinline!y! of another, and the system of equations involve both of them.
+Replacing \lstinline!a! by \lstinline!x-z! (and simplifying the equations) solves this issue and gives the next example.
```

Comment:
The next example?

Reply:
Will reformulate.

Reply:
Have reformulated, the point was that the two examples were related in that way.

Reply:
Good.  Resolving.